articles.tex

```diff
@@ -98,7 +98,8 @@
 A quorum of fifty percent of the Total Number of Possible Votes is required for passage.
 
 \asubsection{Semantic Changes}
-Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
+Any semantic change to the Constitution requires the change to be proposed in writing at a House Meeting.  
+Discussion of the amendment's wording and implications will be held imeadiately following the amendment's proposal.
```

Comment:
"imeadiately" -> "immediately"
"will be held" -> "should be held"

I would also change the wording of this to "Discussion of the wording and implications of the change [...]" as we don't use the word "amendment" here.